packages/atlas/src/components/Fee/Fee.tsx

```diff
@@ -49,6 +49,7 @@ export const Fee: FC<FeeProps> = ({
             variant={variant}
             color={loading ? 'colorTextMuted' : color}
             withTooltip
+            withDenomination="after"
             format="short"
             margin={{ right: 1 }}
           />
```

Comment:
In the action bar:
- The color of dollars is different. It's always `Muted`
- There should be a tooltip for dollars as well
![image](https://github.com/Joystream/atlas/assets/51168865/6388b3bb-33ff-4f8b-a641-9d608e7409f5)
- The spacing between eee and the Information component is incorrect on smaller devices 
![image](https://github.com/Joystream/atlas/assets/51168865/3b783920-e76d-4334-a3f7-be5d85d68a11)
![image](https://github.com/Joystream/atlas/assets/51168865/a92a8f50-fd67-409a-8cc4-5cc7cf8dfd1e)

Reply:
This problem is only valid for Galaxy Fold phones, which we don't really support (for this device width most of the UI is broken, starting with an empty homepage)

Reply:
> Spacing problem is only valid for Galaxy Fold phones, which we don't really support (for this device width most of the UI is broken, starting with an empty homepage)

oh, you're right, my bad. We don't support viewports below 320px.